src/signing/stronghold.rs

```diff
@@ -0,0 +1,69 @@
+use crate::account::{account_id_to_stronghold_record_id, Account};
+
+use std::convert::TryInto;
+
+#[derive(Default)]
+pub struct StrongholdSigner;
+
+impl super::Signer for StrongholdSigner {
+  fn init_account(&self, account: &Account, mnemonic: Option<String>) -> crate::Result<String> {
+    let stronghold_account_res: crate::Result<stronghold::Account> =
+      crate::with_stronghold_from_path(account.storage_path(), |stronghold| {
+        let created_at_timestamp: u128 = account.created_at().timestamp().try_into().unwrap(); // safe to unwrap since it's > 0;
+        let account = match mnemonic {
+          Some(mnemonic) => stronghold.account_import(
+            Some(created_at_timestamp),
+            Some(created_at_timestamp),
+            mnemonic,
+            Some("password"),
```

Comment:
I lack overall knowledge of wallet.rs so: what is this ?

Reply:
do not worry about this, it's a dummy mess we had with the old stronghold :P